researchstudio-sat/webofneeds
# Fix websocket and accesscontrol

Allows websocket connection now.
How to test:
* go to matchat.org, hit F12 to open the console, switch to the network tab and observe that sockjs falls back to XHR after connecting via websocket fails
* go to integrationtest (deploy this branch there if necessary) and observe that sockjs does not need to fall back any more

Then verify that this change does not affect access control by forcing the bot to download the connection data:
* Open a connection to the debugbot
* chat a little
* type `validate` and wait for the result. 
* if the bot finds that the data is valid, we're done 

Fixes #1133
